blaze/expr/expressions.py

```diff
@@ -415,6 +415,17 @@ def sliceit(child, index):
 
 
 class Slice(Expr):
+    """
+    Elements `start` until `stop`
```

Comment:
`Slice` expressions also accept a `step` parameter on many backends. Please include that in the docstring here.